code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -266,7 +266,7 @@
 					CHECK_TICK
 
 /obj/structure/destructible/clockwork/massive/celestial_gateway/attack_ghost(mob/user)
-	if(!user.client || !user.client.holder)
+	if(!user.client || !user.client.holder || !IsAdminGhost(user))
```

Comment:
```user.client.holder``` does this though, right?

Reply:
ITS A FUCKING RIPPIE ISSUE, NOT A /TG/ ISSUE